--- common-password-dpkg	2019-05-23 22:45:33.070077025 +0000
+++ common-password	2019-05-23 22:45:44.770034361 +0000
@@ -23,7 +23,7 @@
 
 # here are the per-package modules (the "Primary" block)
 password	[success=2 default=ignore]	pam_unix.so obscure sha512
-password	[success=1 user_unknown=ignore default=die]	pam_ldap.so use_authtok try_first_pass
+password	[success=1 user_unknown=ignore default=die]	pam_ldap.so try_first_pass
 # here's the fallback if no module succeeds
 password	requisite			pam_deny.so
 # prime the stack with a positive return value if there isn't one already;
